test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -358,14 +358,14 @@ describe('acceptSession()', () => {
     jest.spyOn(mediaUtils, 'createUniqueAudioMediaElement').mockReturnValue(mockAudioElement);
 
     jest.spyOn(BaseSessionHandler.prototype, 'acceptSession');
-    jest.spyOn(mediaUtils, 'attachAudioMedia').mockImplementation((_sdk, _stream, _volume, element) => element);
+    jest.spyOn(mediaUtils, 'attachAudioMedia').mockImplementation((_sdk, _stream, _volume, element) => element as HTMLAudioElement);
     jest.spyOn(handler, 'addMediaToSession').mockImplementation();
 
     const createdStream = new MockStream({ audio: true });
     jest.spyOn(mockSdk.media, 'startMedia').mockResolvedValue(createdStream as any);
 
     const mockIncomingStream = new MockStream({ audio: true });
-    mockSdk._config.defaults.audioVolume = 100;
+    mockSdk._config.defaults!.audioVolume = 100;
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    mockSdk._config.defaults?.audioVolume = 100;
```